### ISSUE TITLE
empty lines in output

### ISSUE THREAD

**Issue by janxkoci** [alanrogers/legofit#9]
Hi Alan, I noticed a _very minor_ issue with most of the tools in your package - blank lines in headers of their outputs.

For example consider this _modified_ example from [your docs](https://alanrogers.github.io/legofit/html/bepe.html):

```
################################################
# bepe: bootstrap estimate of predictive error #
#                 version 1.80                 #
################################################
    < blank line >
# Program was compiled: Feb  8 2019 15:26:39
# Program was run: Fri Feb  8 15:31:50 2019
    < blank line >
#          bepe        DataFile     LegofitFile
1.908861378e-07        sim0.opf    a2-0.legofit
2.495970126e-07        sim1.opf    a2-1.legofit
2.627267214e-07       sim10.opf   a2-10.legofit
 1.71297571e-07       sim11.opf   a2-11.legofit
1.917256551e-07       sim12.opf   a2-12.legofit
```

When processing these files, one usually starts with something like `grep -v ^# file` to filter out the header and only capture data, but this results in two empty lines at the top of the output. To get rid of these, additional pipe through something like `grep .` is needed (depending on what you are doing).

This issue is really minor, just an inconvenience. But it has a simple solution. You probably want to keep the blank lines for spacing, but you can just put the sharp `#` symbol at the beginning of these lines, so they get also captured by filters such as `grep -v ^#`.

Thanks for considering this quality-of-life tweak.

**Comment by alanrogers** [comment]
I have fixed this, at least for legofit, legosim, bepe, booma, and resid. Let me know if you need the same changes in any other programs. The fixed version is 2.3.5 on the devlp branch, which will be merged into master within a couple of weeks.

**Comment by janxkoci** [comment]
Awesome :) I think the python tools don't have this problem and I can't think of another C tool where I would see this, other than those you mention.
